addons/rating/views/rating_templates.xml

```diff
@@ -16,6 +16,35 @@
             </t>
         </template>
 
+        <!-- External page: Invalid partner rating -->
+        <template id="rating_external_page_invalid_partner" name="Not allows to rating">
+            <t t-call="web.frontend_layout">
+                <section class="mt128">
+                    <div class="container">
+                        <div class="row" >
+                            <div class="col-lg-4">
+                                <img class="img img-fluid mx-auto" src="/rating/static/src/img/503.svg" alt=""/>
```

Comment:
Can't we re-use an existing image while we are at it ? Don't we have any 503 ?

Reply:
Hello @tde-banana-odoo , 
No, we haven't any 503 image.